### ISSUE TITLE
Marzipano Tool

### ISSUE THREAD

**Issue by lazareff** [google/marzipano#383]
How can I prepare a set of files without using the Marzipano Tool?
I'm just worried that we will create functionality based on your script, and in the future the Marzipano Tool will stop working :)

**Comment by jrjdavidson** [comment]
I agree with this - it is quite worrisome. I believe the tool is entirely browser based and all processing is done in the browser.. is the source available anywhere ?

**Comment by israelbar** [comment]
any news on this?

**Comment by mkubdev** [comment]
Sup guys, do you want to collaborate on an opensource solution to generate tiles from 2:1 panorama image ?

**Comment by jrjdavidson** [comment]
I wouldn't be much help, but keen to do what I can
